docs/index.rst

```diff
@@ -130,7 +130,8 @@ Contents
     contributing
     release
     license
-
+    Official Website<https://zarr.dev/>
```

Comment:
Maybe something like this?

```suggestion
    `Home page <https://zarr.dev/>`_
```

Reply:
@jakirkham I just updated the PR with the change you requested